docs/labs/installer.rst

```diff
@@ -57,133 +89,179 @@ Make a local directory for your cluster files
+    If necessary, generate and download a new keypair in the EC2 console.
 
-.. warning::
-    Make sure you can log into the nodes as **root** with a private key (e.g. on Ubuntu or CoreOS on AWS, log in as the default user, then run ``sudo mkdir /root/.ssh; sudo cp .ssh/authorized_keys /root/.ssh/authorized_keys``).
+Now paste the following variables into your ``terraform.tfvars`` file::
 
-You may want to pick a node to be the control node and give it a DNS name (if you do this, set up an A record for it with your DNS provider).
-Using a DNS name is optional, you can also just use its IP address.
+    # AWS keys
+    aws_access_key = "your AWS access key"
+    aws_secret_key = "your AWS secret key"
 
-cluster.yml
-===========
+    # AWS region and zone
+    aws_region = "region you want nodes deployed in e.g. us-east-1"
+    aws_availability_zone = "zone you want nodes deployed in e.g. us-east-1a"
 
-Run the following command in your ``~/clusters/test`` directory you made earlier:
+    # Key to authenticate to nodes via SSH
+    aws_key_name = "name of EC2 keypair"
+    private_key_path = "absolute path to EC2 key (.pem file) on your local machine"
+
+    # Instance types and number of nodes (total launched = agent_nodes + 1, for master)
+    aws_instance_type = "m3.large"
+    agent_nodes = "2"
+
+.. note::
+
+    By default, the installer will launch one master node (where the control service runs) and two agent nodes (where volumes get attached and containers run).
+    Please refer to the `AWS pricing guide <https://aws.amazon.com/ec2/pricing/>`_ to understand how much this will cost you.
+
+Now run the following command to automatically provision some nodes.
 
 .. prompt:: bash $
 
-    flocker-sample-files
+    uft-flocker-sample-files
+    uft-flocker-get-nodes --ubuntu-aws
 
-This will create some sample configuration files that correspond to the backend Flocker will use - base your ``cluster.yml`` on one of these files:
+This step should take 30-40 seconds, and then you should see output like this::
 
-* AWS EBS: ``cluster.yml.ebs.sample``
-* OpenStack (including Rackspace): ``cluster.yml.openstack.sample``
-* ZFS (local storage): ``cluster.yml.zfs.sample``
+    Apply complete! Resources: 10 added, 0 changed, 0 destroyed.
 
-.. warning::
-    Note that ZFS support is experimental, and should not be used for production workloads.
+This should have created a pre-configured ``cluster.yml`` file in the current directory.
 
-Choose the one that's appropriate for you, and then customize it with your choice of text editor.
-For example:
+Now you have some nodes, it's time to install and configure Flocker on them!
+
+Install and Configure Flocker
+=============================
+
+Run the following command:
 
 .. prompt:: bash $
 
-    mv cluster.yml.ebs.sample cluster.yml
-    vim cluster.yml # customize for your cluster
+    uft-flocker-install cluster.yml && uft-flocker-config cluster.yml && uft-flocker-plugin-install cluster.yml
```

Comment:
The path with `/host` in it is the path that exists inside the UFT container. That means that *outside* the UFT container `/Users/madhuriyechuri/hybrid-master.pem` needs to exist. However, it looks like you're running this from inside an Ubuntu VM. If that's the case, you need to specify `private_key_path` as the absolute path to the key inside the VM...

FWIW, this is designed to be seamless if you're using Docker Toolbox, which the official Docker instructions now recommend. That uses `docker-machine` to spin up a VirtualBox instance with `boot2docker` installed on it, with host-path sharing set up.

Can you describe whether you've set up any host path sharing for your Vagrant Ubuntu VM?